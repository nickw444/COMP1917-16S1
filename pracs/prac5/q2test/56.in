56
8 -3 6 -2 -5 0 -7 -9 -2 -3 -9 -1 9 -8 3 -5 0 -3 -1 -9 -8 -6 -4 7 -8 2 -10 8 -6 -5 6 -8 -8 1 9 -3 -3 4 -5 4 8 -1 6 0 10 0 -2 -4 7 0 9 9 -6 -10 -9 2
