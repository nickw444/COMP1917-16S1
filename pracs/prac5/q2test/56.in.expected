average of array: -0.982143
